iphone/Classes/TiUIListView.m

```diff
@@ -1749,6 +1749,11 @@ - (CGFloat)tableView:(UITableView *)tableView heightForHeaderInSection:(NSIntege
     if (size < DEFAULT_SECTION_HEADERFOOTER_HEIGHT) {
       size += DEFAULT_SECTION_HEADERFOOTER_HEIGHT;
     }
+  } else if (0 == section) {
+    // Make sure grouped style always shows the 1st header if it contains no text/view.
+    if ((tableView.style == UITableViewStyleGrouped) || (tableView.style == UITableViewStyleInsetGrouped)) {
+      size = 35.0;
```

Comment:
I don't feel well with static sizes, could this maybe be received other way?

Reply:
I would if I knew how.

Note that this code will only be used if the 1st section has no text/views and for grouped mode only. So, it's a fallback.

Reply:
After some testing with a native project it seems like it is the default behavior that the header view is not displayed if it is empty. Although i'm not sure why it renders the header view initially, handling this edge case with a hardcoded value seems to be the only viable solution for now.

Speaking of the hardcoded value, where did you get `35.0` from? I see there already is a `DEFAULT_SECTION_HEADERFOOTER_HEIGHT` constant defined in the code. Why not use that?

Reply:
The `DEFAULT_SECTION_HEADERFOOTER_HEIGHT` is `29.0` which does not match Apple's "grouped" section height. I've compared screenshots and looked at the pixels. Apple is using a 35 point height. Others on stackoverflow have noted the same for the past several years. I wish I could grab the height via an API, but I haven't seen anything available. If you can think of a better alternative, then I'm all ears.